python/pyspark/sql/readwriter.py

```diff
@@ -297,7 +300,7 @@ def text(self, paths):
     def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=None,
             comment=None, header=None, inferSchema=None, ignoreLeadingWhiteSpace=None,
             ignoreTrailingWhiteSpace=None, nullValue=None, nanValue=None, positiveInf=None,
-            negativeInf=None, dateFormat=None, timestampFormat=None, maxColumns=None,
+            negativeInf=None, dateFormat=None, timestampFormat=None, timeZone=None, maxColumns=None,
```

Comment:
(Hi @ueshin, up to my knowledge, this should be added at the end to prevent breaking the existing codes that use those options by positional arguments)